Add jenkinsfile

angelakis opened the pull request:
@nsklikas

angelakis committed:
Add jenkinsfile and Dockerfile for build

nsklikas commented:
Let's add `--deselect tests/test_51_client.py::TestClient::test_create_auth_request_requested_attributes` to the pytest call and merge this

nsklikas committed:
Remove failing test

This test was failing because Jenkins runs on a different filesystem than expected which
causes the attribute maps to be parsed in an unexpected order. This is a bug in the
tests
